packages/libs/react-ui/src/components/ProgressBar/ProgressBar.css.ts

```diff
@@ -94,26 +107,47 @@ export const circleStyle = style([
   }),
 ]);
 
-export const lineStyle = style([
+export const firstCircleStyle = style([
   sprinkles({
-    borderRadius: '$sm',
-    height: '$2',
+    paddingTop: '$1',
+  }),
+]);
+
+export const lastCircleStyle = style([
+  sprinkles({
+    paddingBottom: '$1',
```

Comment:
You can add both of these to the main circleStyle style implementation and apply styles depending on if they are the first-child or last-child